plugins/video/pkg_src/src/decoder/video_decoder_mixed.h

```diff
@@ -29,7 +29,13 @@ class VideoDecoderMixed : public dali::Operator<dali::MixedBackend> {
  public:
   explicit VideoDecoderMixed(const dali::OpSpec &spec)
     : Operator<dali::MixedBackend>(spec)
-    , device_id_(spec.GetArgument<int>("device_id")) {}
+    , device_id_(spec.GetArgument<int>("device_id")) {
+      if (spec.HasArgument("end_frame")) {
+        end_frame_ = spec.GetArgument<int>("end_frame");
+      } else {
+        DALI_FAIL("Not passing `end_frame` argument is not yet implemented.");
```

Comment:
Nitpick, to get rid two negations from the sentence.

```suggestion
        DALI_FAIL("Currently, `end_frame` argument is required.");
```

Reply:
Done